sdk/cmake/set_wine_module.cmake

```diff
@@ -0,0 +1,7 @@
+
+function(set_wine_module TARGET)
+    target_include_directories(${TARGET} BEFORE PRIVATE
+        ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine
+        ${REACTOS_BINARY_DIR}/sdk/include/reactos/wine)
+    target_compile_definitions(${TARGET} PRIVATE __WINESRC__) # FIXME: CORE-5743: Delete this line
```

Comment:
My suggestion would be to create up to 3 functions and use them to get code equality, first (commit):
* `set_wine_module()`: `target_include_directories()` only, the goal.
* `set_wine_moduleW()`: `target_compile_definitions()` only, if there are such cases.
* `set_wine_module2()`: both, as most current uses seem to be.

Then (in this PR or a follow-up(s), you could look into what to do to (step by step) get rid of the extra function(s).

Reply:
`set_wine_module_FIXME_1` and `set_wine_module_FIXME_2` added.